frontend/src/container/AppLayout/index.tsx

```diff
@@ -279,6 +278,14 @@ function AppLayout(props: AppLayoutProps): JSX.Element {
 
 	const isSideNavCollapsed = getLocalStorageKey(IS_SIDEBAR_COLLAPSED);
 
+	/**
+	 * Note: Right now we don't have a page-level method to pass the sidebar collapse state.
+	 * Since the use case for overriding is not widely needed, we are setting it here
+	 * so that the workspace locked page will have an expanded sidebar regardless of how users
+	 * have set it or what is stored in localStorage. This will not affect the localStorage config.
+	 */
+	const isWorkspaceLocked = (): boolean => pathname === ROUTES.WORKSPACE_LOCKED;
```

Comment:
why is this still a function ? 

```const isWorkspaceLocked = (pathname === ROUTES.WORKSPACE_LOCKED);```

Reply:
@vikrantgupta25, I followed our conversation (recording) and used the same approach. I will change it to const `isWorkspaceLocked = (pathname === ROUTES.WORKSPACE_LOCKED);`. Thank you!